design-system/StyledContainer.js

```diff
@@ -16,4 +14,9 @@ export const StyledContainer = styled.div`
     $isSpaceBetween
       ? `flex-direction:row; align-items: center; justify-content: space-between; height:auto;`
       : ``};
+
+  ${({ $isCenter }) =>
+    $isCenter
+      ? `flex-direction:row;align-items:center;justify-content:center;`
+      : ``};
```

Comment:
I copied this from your refactoring PR:

I recommend using the `css` helper (don't forget to import it) if you want to set several properties based on a prop.

```suggestion
  ${({ $isCenter }) =>
    $isCenter &&
    css`
      flex-direction: row;
      align-items: center; 
      justify-content: center;
      `};
```

This makes your code a lot more readable.

See the [handout](https://github.com/neuefische/hh-web-23-3/blob/main/sessions/react-styled-components-2/react-styled-components-2.md#adapting-based-on-props) for styled components as reference.